srcs/layouts/latn_qwerty_lt.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard name="QWERTY (Lietuviškai)" script="latin">
+  <row>
+    <key shift="0.5" key0="ą"/>
+    <key key0="č"/>
+    <key key0="ę"/>
+    <key key0="ė"/>
+    <key key0="į"/>
+    <key key0="š"/>
+    <key key0="ų"/>
+    <key key0="ū"/>
+    <key key0="ž"/>
```

Comment:
Not having such rows is one of the purpose of this keyboard. All these letters should be swipes on other keys.

Reply:
It's a copy of the traditional Lithuanian layout, used on PC, Samsung, AnySoftKeyboard, and some other keyboards.

Reply:
In this app, symbols in the corner of keys are more easily accessed than on other layouts. It's not useful to copy layouts directly from other apps, instead they should be adapted to make use of the app's features.